### ISSUE TITLE
internal player won't play a video that VLC does

### ISSUE THREAD

**Issue by risacher** [popcorntime/popcorntime#767]
Operating System Version:
OS X 10.13.3

Popcorn Time Version:
0.3.10 - "last successful CI build" as-of 2018-04-23

Download date:
2018-04-23

#### Expected Behaviour
Play video with internal viewer

#### Actual Behaviour
Error screen: "The video could not be loaded, either because the server or network failed, or because the file format is not supported."

#### Steps to reproduce the behaviour
Play Westworld, Season 2, Episode 1, at 720P with the internal player.

Note: downloading the torrent from the magnet link works.  Playing the resultant video in VLC works fine.  Streaming to VLC from PT works.  But the internal viewer does not work.  Ran nwjs from command line and on error the terminal prints this additional error:

`[21194:775:0423/024041:ERROR:render_media_log.cc(25)] MediaEvent: PIPELINE_ERROR decoder: not supported`

**Comment by ozkanpakdil** [comment]
while I was trying to upgrade nwjs I have seen this too. and I tried to solve it but looks like popcorn lost a developer who suppose to compile ffmpeg properly. then I reverted my changes back. tried to apply https://github.com/iteufel/nwjs-ffmpeg-prebuilt/ but it did not work for me in linux. I wonder would it work for OSX :smile: 

if it does not work you can always compile it and replace libraries. but I must say it takes long.

**Comment by Persei08** [comment]
@ozkanpakdil , I think https://github.com/butterproject/nwjs-build is the script used to compile the custom nwjs for linux x64 (haven't tried myself)

**Comment by ozkanpakdil** [comment]
yep @Persei08 I used that one. I was thinking to compile and actually replace with newer version of nwjs. after 1 day compile time I did for linux. then I realize this is only linux and in my local I needed to change some python build script :) because of lld, clang was giving errors in linux mint. 

long story short I reverted back. nwjs compile was more complicated then what I was expecting. maybe if nwjs supports all codecs in the future I can upgrade it. until then stick with https://get.popcorntime.sh/repo/nw/
